src/slurm_plugin/clustermgtd.py

```diff
@@ -300,6 +313,8 @@ def set_config(self, config):
             self._config = config
             self._compute_fleet_status_manager = self._initialize_compute_fleet_status_manager(config)
             self._instance_manager = self._initialize_instance_manager(config)
+            self._disable_protected_mode = self._is_protected_mode_disabled(config)
+            self._partitions_protected_failure_count_map = self._initialize_partitions_protected_failure_count()
```

Comment:
Do we need to restart clustermgtd to apply the new config after we changing the config? Is it after the clustermgtd restart, the protected failure count will be clear, so it will reset failure count and keep count again?